.github/workflows/benchmarks.yaml

```diff
@@ -3,6 +3,10 @@ on:
   push:
     branches:
     - '**'
+    paths-ignore:
+      - 'docs/**'
+      - 'keps/**'
+      - '*.md'
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths

> When all the path names match patterns in paths-ignore, the workflow will not run. If any path names do not match patterns in paths-ignore, even if some path names match the patterns, the workflow will run.